Library/Navigation.swift

```diff
@@ -47,7 +47,7 @@ public enum Navigation: Equatable {
     case pledge(Navigation.Project.Pledge)
     case updates
     case update(Int, Navigation.Project.Update)
-    case survey(Int)
+    case survey(Int, String)
```

Comment:
Are both guaranteed to be set? I thought in this case the surveyParam was `0`.

Reply:
If not, it might make sense to split this enum into a type for survey-with-id and survey-with-url.